### PR TITLE
Support tf tf repo macros in external repos.

### PR DESCRIPTION
**Pull request by stgpetrovic**
Support tf tf repo macros in external repos.

Using `//tensorflow` in a macro makes it unusable in repote repos; using `clean_dep` which is basically `str(label($))` makes it work from wherever it's included.

### CODE REVIEW

**Comment by lsy323** [comment]
Thanks @stgpetrovic! Minor question: why is `bazelisk-linux-amd64` added?

cc @JackCaoG This PR fixes using local TF repo in PT/XLA build. Following https://github.com/pytorch/xla/blob/5b5dd27f20ec80706e2d88f59b1826a5924c66f2/WORKSPACE#L33 to use local TF should work after this PR.

**Comment by stgpetrovic** [comment]
Removed the binary.

**Review by JackCaoG — APPROVED** [review]

**Review by lsy323 — APPROVED** [review]
LGTM
